### PR TITLE
Clowns can buy the Chainsaw as a Traitor

### PR DESCRIPTION
**Pull request by TheOneAndOnlyCreeperJoe**
Come on, everyone wants it.

### CODE REVIEW

**Comment by Crystalwarrior** [comment]
add the mime and the entire civvie division to this.

**Comment by Pyko1** [comment]
chainsaw op, and too many civvies.

Also give the clown a complimentary weldingmask if he buys this.

**Comment by Kierany9** [comment]
We might as well give it to the entire civvie department(except cargo but i dont think they count). It'd make sense not to do it if it was botany-exclusive but the chef and bartender get it too for whatever reason.

**Comment by ghost** [comment]
To be honest, I always thought the chainsaw should be botanist only.

**Comment by Crystalwarrior** [comment]
But it's too fun to be botanist-only :(

**Comment by Carbonhell** [comment]
Christ no, the current thing + clowns is fine, don't give it to everyone are you crazy.
Code's fine,by the way

**Comment by trifornt-tinfornt** [comment]
I agree, the Chainsaw is at its best when you're the Bartender or something so even then as a Botanist/Chef/Bartender exclusive there's only really one job where it would shine. Also the Clown using something like the Chainsaw is just fitting. Not this garbage, Joker-ass "Cane Shotgun" that's horribly overpriced when you can just buy the toxins kit and do it yourself.

**Comment by Nebyab** [comment]
no, we really really really really don't need the clown to have the generic chainsaw.

Give the clown its own clown themed chainsaw which is identical stat-wise, but not spritewise.

If you can't come up with a fun enough sprite for it, then no.

**Comment by dreamcrab** [comment]
i also enjoy the idea of a clownsaw

**Comment by Carbonhell** [comment]
don't forget to put a smile on it

**Comment by trifornt-tinfornt** [comment]
Clownsaw absolutely. Nebyab has the right idea.
Get to spriting, Joe.

**Comment by calibraptor** [comment]
+1 for clownsaw

**Comment by Lawligagger** [comment]
yes pls

**Comment by TheOneAndOnlyCreeperJoe** [comment]
ehh sure I'll give the clown his own chainsaw

**Comment by TheOneAndOnlyCreeperJoe** [comment]
Preview of what it is RN:
http://i.imgur.com/4jrm5aD.gifv

**Comment by trifornt-tinfornt** [comment]
Not bad, but I think the Clownsaw should be kind of a red chainsaw with splotches of white and yellow on it - make it look kind of "circus-y."
(that also would have been easier to make than that good lord how much time did you put into that)

File: code/datums/uplink_item.dm [hunk 1]
```diff
@@ -150,7 +150,7 @@ var/list/uplink_items = list()
 	desc = "An extremely loud, dirty, noisy, bulky, powerful as hell chainsaw that will absolutely destroy anyone it comes in contact with. Obviously won't fit in your backpack."
 	item = /obj/item/weapon/twohanded/chainsaw
 	cost = 14
-	jobs = list("Botanist", "Bartender", "Chef")
+	jobs = list("Botanist", "Bartender", "Chef", "Clown")
```
Comment (TDcoolguy300) [inline code/datums/uplink_item.dm:153]: Why do clowns get both chainsaws?

Comment (TheOneAndOnlyCreeperJoe) [inline code/datums/uplink_item.dm:153]: Oops, oversight

**Comment by TheOneAndOnlyCreeperJoe** [comment]
I'll redo this shit once I finished the sprite. can't be arsed to solve the conflict either.

**Pull request closed without merging**